src/telemetry/telemetryService.ts

```diff
@@ -200,3 +235,30 @@ export class TelemetryService {
     { leading: true, trailing: false }
   );
 }
+
+/**
+ * TODO: This hashing function should be moved to devtools-shared.
```

Comment:
Can we make this `TODO(VSCODE-XXX)` with a tracking ticket? :)

Reply:
my lack of desire to open Jira motivated https://github.com/mongodb-js/devtools-shared/pull/532